src/modules/forms.css

```diff
@@ -49,10 +40,9 @@ textarea {
   background-color: var(--baseBackgroundColor);
   border: var(--baseBorder);
   border-radius: var(--baseBorderRadius);
-  box-shadow: var(--formBoxShadow);
```

Comment:
same, just change the value of the --formBoxShadow variable instead of removing the property